x/bank/keeper/keeper.go

```diff
@@ -59,6 +59,7 @@ type BaseKeeper struct {
 	storeKey               storetypes.StoreKey
 	paramSpace             paramtypes.Subspace
 	mintCoinsRestrictionFn MintingRestrictionFn
+	authority  			   string
```

Comment:
The `NewBaseKeeper` func should be updated to take in and set this value. When calling it, you provide the gov module's address for it (similar to the `x/upgrade` module).